packages/bsky/src/api/app/bsky/unspecced/getTimelineSkeleton.ts

```diff
@@ -48,7 +40,23 @@ export default function (server: Server, ctx: AppContext) {
             ref('originatorDid'),
           ]),
         )
+        .selectAll('feed_item')
+
+      const selfBuilder = ctx.db.db
+        .selectFrom('feed_item')
+        .where('feed_item.originatorDid', '=', viewer)
+        .selectAll('feed_item')
```

Comment:
Only possible (?) snag I could see here is viewing your own reposts of posts by someone who blocked you.

Reply:
eh good call - i'll just add in all the mute & block checks for consistencies sake. they're pretty cheap